File: PRAK402-2310817220016-NailaHanifah.c
#include <stdio.h>

int main() {
    int batas;

    printf("Input (batas maksimal): ");
    scanf("%d", &batas);

    for (int i = 1; i <= batas; i += 2) {
        printf("%d ", i);
    }
    printf("\n");

    for (int i = batas % 2 == 0 ? batas : batas - 1; i >= 2; i -= 2) {
        printf("%d ", i);
    }
    return 0;
}

File: PRAK402-2310817220016-NailaHanifah.py
batas = int(input("Input (batas maksimal): "))

for i in range(1, batas+1, 2):
    print(i, end=" ")
print()

for i in range(batas if batas % 2 == 0 else batas - 1, 1, -2):
    print(i, end=" ")